src/components/UserInfo/UserInfo.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Users } from '../types';
+
+interface Props {
+  user?: Users;
```

Comment:
make this prop required. Your userInfo always should get user. And you will just render this component conditionally if user exists